apps/wfo-ui-surf/components/WfoServiceTicketsList/WfoServiceTicketsList.tsx

```diff
@@ -124,24 +122,24 @@ export const WfoServiceTicketsList = ({
             field: SERVICE_TICKET_FIELD_START_DATE,
             name: t('startDate'),
             width: '150',
-            render: (date) => <WfoDateTime dateOrIsoString={date} />,
-            renderDetails: parseIsoString(parseDateToLocaleDateTimeString),
-            clipboardText: parseIsoString(parseDateToLocaleDateTimeString),
+            render: (date) => (
+                <span>{parseDateToLocaleDateTimeString(parseDate(date))}</span>
```

Comment:
if the span is not required you can omit that. Also there is another utility function that can do what you are doing here:

packages/orchestrator-ui-components/src/utils/date.ts --> formatDate

Line 125-127 (the render function) can be replaced with:
`render: formatDate,`

Same applies to all renders of dates here.